Assets/UI/Scripts/OptionsGraphics.gd

```diff
@@ -0,0 +1,33 @@
+extends GridContainer
+
+func _ready():
+	$NameEdit.text = Global.player_name
+	# add language from Global
+	var idx = 0
+	var i = 0
+	for n in Global._lang_array:
+		$LanguageEdit.add_item(Global._languages[n])
+		if n == Global.language:
+			idx = i
+		i += 1
+	$LanguageEdit.text = Global._languages[Global.language]
+	$LanguageEdit.select(idx)
```

Comment:
I know this is mostly placeholder, but the name and language should not be in the "Graphics" section. I suggest moving these to the "Game" section, and also make "Game" the left-most tab.

Also, if you can, the UI should be in a similar style to the new game menu.

Reply:
What do you think of it now?